[2025-03-14T13:29:36.376+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T13:29:36.390+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mtrix_clientes_edi.ingest_clientes_mtrix manual__2025-03-14T13:23:18.201399+00:00 [queued]>
[2025-03-14T13:29:36.397+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mtrix_clientes_edi.ingest_clientes_mtrix manual__2025-03-14T13:23:18.201399+00:00 [queued]>
[2025-03-14T13:29:36.397+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 2
[2025-03-14T13:29:36.409+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): ingest_clientes_mtrix> on 2025-03-14 13:23:18.201399+00:00
[2025-03-14T13:29:36.417+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=72) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-14T13:29:36.419+0000] {standard_task_runner.py:72} INFO - Started process 74 to run task
[2025-03-14T13:29:36.419+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'mtrix_clientes_edi', 'ingest_clientes_mtrix', 'manual__2025-03-14T13:23:18.201399+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/mtrix_clientes_edi.py', '--cfg-path', '/tmp/tmpa8lc0162']
[2025-03-14T13:29:36.421+0000] {standard_task_runner.py:105} INFO - Job 12: Subtask ingest_clientes_mtrix
[2025-03-14T13:29:36.596+0000] {task_command.py:467} INFO - Running <TaskInstance: mtrix_clientes_edi.ingest_clientes_mtrix manual__2025-03-14T13:23:18.201399+00:00 [running]> on host e1b30e32018f
[2025-03-14T13:29:36.653+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mtrix_clientes_edi' AIRFLOW_CTX_TASK_ID='ingest_clientes_mtrix' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T13:23:18.201399+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-14T13:23:18.201399+00:00'
[2025-03-14T13:29:36.654+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-14T13:29:36.732+0000] {logging_mixin.py:190} INFO - Arquivo 'CLIENTESUN12032025140133.txt' processado com sucesso.
[2025-03-14T13:29:36.733+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-14T13:29:36.741+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-14T13:29:36.742+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=mtrix_clientes_edi, task_id=ingest_clientes_mtrix, run_id=manual__2025-03-14T13:23:18.201399+00:00, execution_date=20250314T132318, start_date=20250314T132936, end_date=20250314T132936
[2025-03-14T13:29:36.793+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-14T13:29:36.809+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-14T13:29:36.810+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
